Add Nontarget highlight orange

jpalmerpivotal opened the pull request:
Add Nontarget highlight orange
Completely untested just editted in github raw

jpalmerpivotal committed:
Add nontarget to model blindly

jpalmerpivotal committed:
non target css

jpalmerpivotal committed:
non target view

ericpromislow commented on build/view.js:
```diff
@@ -10,6 +10,7 @@ export default function View() {
     }
     this.model = null;
     this.wordIsInvalid = false;
+    this.wordIsNonTarget - false;
```
Should be `this.wordIsNonTarget = false;`

ericpromislow reviewed (changes requested):
Found a tyop

Commit:
Made some corrections

Commit:
On a secondary word, just show a prompt.

Don't bother coloring the letters.

One of the advantages of this app is that localizing isn't an issue,
and it's easier to show a line of text than try to express something
through color or sound (and nighttime lirdle players asked for no
sounds).

Commit:
Make it fit in an iphone dimension.

ericpromislow reviewed (approved):
let's do it
